lib/codecs/adts.js

```diff
@@ -75,8 +75,10 @@ AdtsStream = function(handlePartialSegments) {
 
     // unpack any ADTS frames which have been fully received
     // for details on the ADTS header, see http://wiki.multimedia.cx/index.php?title=ADTS
-    while (i + 5 < buffer.length) {
 
+    // we use i + 7 here because we want to be able to parse the entire header.
+    // If we don't have enough bytes to do that, than we definitely won't have a full frame.
```

Comment:
```suggestion
    // We use i + 7 here because we want to be able to parse the entire header.
    // If we don't have enough bytes to do that, then we definitely won't have a full frame.
```